src/components/projects/create-project-form/create-project-form.tsx

```diff
@@ -23,23 +23,26 @@ const CreateProjectForm: React.FC = () => {
               <Row>
                 <Col>
                   <Row>
-                    <Form.Group as={Col} aria-required>
+                    <Form.Group as={Col} controlId='project-name-group' aria-required>
                       <Form.Label htmlFor='project-name'>Project Name</Form.Label>
                       <Form.Control id='project-name' type='text' placeholder='Enter project name...' required />
+                      <Form.Control.Feedback type='invalid'>Please provide a project name.</Form.Control.Feedback>
                     </Form.Group>
                   </Row>
                   <Row>
-                    <Form.Group as={Col} aria-required>
+                    <Form.Group as={Col} controlId='car-number-group' aria-required>
                       <Form.Label htmlFor='car-number'>Car Number</Form.Label>
-                      <Form.Control id='car-number' type='text' placeholder='Enter car number...' required />
+                      <Form.Control id='car-number' type='number' min={1} placeholder='Enter car number...' required />
```

Comment:
Let's set this min to 0 pls. 0 allows for projects unattached to a car. Something i wanna give PMO the flexibility to decide later how to use. Potentially big business projects we want to experiment tracking, or other things.